community/samples/serving/helloworld-java-quarkus/Dockerfile.native

```diff
@@ -0,0 +1,16 @@
+FROM quay.io/rhdevelopers/quarkus-java-builder:graal-1.0.0-rc15 as builder
```

Comment:
Non-blocking nit: Some IDEs recognize *.Dockerfile as a Dockerfile but not Dockerfile.*.